main.js

```diff
@@ -0,0 +1,89 @@
+const menu = document.querySelector(".menu");
+const menuItems = document.querySelectorAll(".menuItem");
+const hamburger = document.querySelector(".hamburger-btn");
+const closeIcon = document.querySelector(".closeIcon");
+const menuIcon = document.querySelector(".menuIcon");
+
+function toggleMenu() {
+  if (menu.classList.contains("showMenu")) {
+    menu.classList.remove("showMenu");
+    closeIcon.style.display = "none";
+    menuIcon.style.display = "block";
+  } else {
+    menu.classList.add("showMenu");
+    closeIcon.style.display = "block";
+    menuIcon.style.display = "none";
+  }
+}
+menu.addEventListener("click", toggleMenu);
+hamburger.addEventListener("click", toggleMenu);
+
+//Form-validation//
+const form = document.getElementById("form");
+
+// Check if String has Uppercase
+function checkUppercase(str) {
+  for (var i = 0; i < str.length; i++) {
+    if (
+      str.charAt(i) == str.charAt(i).toUpperCase() &&
+      str.charAt(i).match(/[a-z]/i)
+    ) {
+      return true;
+    }
+  }
+  return false;
+}
```

Comment:
- [ ] Kindly be noticed that the way you write the `checkUppercase` function will return true if there is at least one lower case character and it is required that the whole email should be in lower case, so you need to write your function like this.

```javascript
function checkUppercase(str) {
  for (var i = 0; i < str.length; i++) {
    if (
      str.charAt(i) != str.charAt(i).toUpperCase() &&
      !str.charAt(i).match(/[a-z]/i)
    ) {
      return false;
    }
  }
  return true;
}
```